src/pages/portfolio/index.tsx

```diff
@@ -77,43 +78,61 @@ const useStyles = makeStyles((theme) => ({
 
 const PortfolioPage: FC<{ data: ProjectGQL }> = ({ data }) => {
   const classes = useStyles();
-  const defaultType = 'All';
   const projectData = data.portfolioPage.frontmatter;
   const { projects } = projectData;
   const { componentType, isMobile } = useComponentType();
   const developerProfile = useDeveloperProfile();
   const [selectedProject, setSelectedProject] = useState(-1);
-  const [navbarTitle, setNavbarTitle] = useState(0);
-  const projectLabels = [defaultType, ...new Set(projects.map((project) => project.projectLabel))];
-
+  const [navbarNumberTitle, setNavbarNumberTitle] = useState(0);
+  const sortedGroupsOfProjects = getTabsData(projects);
+  const projectsLabels: string[] = [...sortedGroupsOfProjects.map((object) => object.projectLabel)];
   const getNavbarTitle = (type: number) => {
-    const title: Record<number, string> = { ...projectLabels };
+    const title = projectsLabels;
     return title[type];
   };
+  const tabLabelTitle = getNavbarTitle(navbarNumberTitle);
 
-  const projectType = getNavbarTitle(navbarTitle);
-
-  const filteredProjects = projects.filter((project) => {
-    return projectType !== defaultType ? project.projectLabel === projectType : ' ';
-  });
+  // filter all projects depending on their label
+  const filteredProjects = sortedGroupsOfProjects
```

Comment:
Maybe just name the variable `projesctFilteredByLabel` and remove the comment above